### ISSUE TITLE
SSE Display Tweaks

### ISSUE THREAD

**Issue by PhantomFlayer** [tiltedphoques/Mod-Compatibility#29]
# Mod information

#### Download link
https://www.nexusmods.com/skyrimspecialedition/mods/34705

#### Dependencies
- SKSE
- Address Library

# Compatibility
- Seems to work fine. Haven't tested an actual connection yet, but the Reborn UI appears and the game functions.

# Additional Information
- Will likely work just fine if only one person connected to a server has it, though you may need to disable the .ini HAVOK tweaks in this scenario, or object physics may not work as intended.

**Comment by androbourne** [comment]
Just wanted to mention this one. It will work as it isn't a true mod file but instead it is a modification to your local game files. You will not see it in the loadorder.txt file because for that reason it should work regardless and not be dependent on if everyone else uses it or not.

This was our experience in group of 4 with mod checker enabled. It wasn't in the load order and half our team had it enabled and the other half did not and they were able to join the server and play with zero issues.

**Comment by PhantomFlayer** [comment]
> Just wanted to mention this one. It will work as it isn't a true mod file but instead it is a modification to your local game files. You will not see it in the loadorder.txt file because for that reason it should work regardless and not be dependent on if everyone else uses it or not.
> 
> This was our experience in group of 4 with mod checker enabled. It wasn't in the load order and half our team had it enabled and the other half did not and they were able to join the server and play with zero issues.

Yes, this is an SKSE plugin. However, much like Engine Fixes, Display Tweaks makes changes to the game's engine, which has the potential to conflict with engine changes that Skyrim Together itself makes - hence the need to test.

**Comment by androbourne** [comment]
Of course. I did not say don't test it. Just providing feedback and status since we of 4 have been using it since day one with zero issues.

**Comment by PhantomGamers** [comment]
I haven't noticed any issues with this either but I don't know enough about the engine changes that ST makes to say for sure if there are any conflicts there.

**Comment by miredirex** [comment]
I use v0.5.15. <kbd>Right Ctrl</kbd> UI is more stable in the <kbd>Esc</kbd> menu, and may stutter or not work in-game. "High performance configuration" for Display Tweaks works fine for me as well
